crates/wit-parser/src/ast.rs

```diff
@@ -549,6 +552,30 @@ enum InterfaceItem<'a> {
     TypeDef(TypeDef<'a>),
     Func(NamedFunc<'a>),
     Use(Use<'a>),
+    Nest(Nest<'a>),
+}
+
+struct Nest<'a> {
+    docs: Docs<'a>,
+    attributes: Vec<Attribute<'a>>,
+    from: UsePath<'a>,
+}
+
+impl<'a> Nest<'a> {
+    fn parse(
+        tokens: &mut Tokenizer<'a>,
+        docs: Docs<'a>,
+        attributes: Vec<Attribute<'a>>,
+    ) -> Result<Self> {
+        tokens.eat(Token::Nest)?;
```

Comment:
I think you'll want to `expect` here instead of `eat` (as `eat` has a boolean return)